packages/eslint-config/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "@xt-pagesource/eslint-config",
+  "version": "1.0.0",
+  "description": "XT PageSource (ESLint) configuration",
+  "keywords": [
+    "eslint",
+    "prettier",
+    "rules",
+    "lint",
+    "javascript"
+  ],
+  "main": "index.js",
+  "repository": "https://github.com/pagesource/quality-control.git",
+  "author": "Sapient XT",
+  "license": "MIT",
+  "files": [
+    "index.js"
+  ],
+  "dependencies": {
```

Comment:
We need to keep them in peer dependencies.
Think about it - the execution will happen where the actual code will be and where the module will be consumed. 

It will be working in your examples because node dependency resolution work bottom up. If you move your example outside the folder, it may not work

Reply:
@ankurkakroo2  done